{{cookiecutter.repo_name}}/.env

```diff
@@ -0,0 +1,15 @@
+
+{% if cookiecutter.stripe %}
+STRIPE_SECRET_KEY="{{ cookiecutter.stripe_key }}"
+STRIPE_PUBLIC_KEY=""
+{%- endif %}
+
+AWS_ACCESS_KEY_ID=""
```

Comment:
Is .env only for local settings? Normally locally we do everything on the harddrive and not remotely